### PR TITLE
Move check versions script + fix a bug

### PR DESCRIPTION
**Pull request by marcofranssen**
See https://github.com/spiffe/helm-charts/pull/236 for the result.

https://github.com/spiffe/helm-charts/actions/runs/4795275892 for the workflow execution.

### CODE REVIEW

**Comment by kfox1111** [comment]
Not seeing any fixes to the job. Just better formatting.

**Comment by faisal-memon** [comment]
> ah. I think I finally understand after doing some debugging. Can we break this pr up into two for posterity (or 2 commits)? Its really hard to follow that there is actually one small fix in here to make sure the new version gets into the json file with it being hidden by all the other changes going on that's just format changes without any functional difference, along with a file rename.

@marcofranssen Or do a move of the file to the new directory instead of delete and create a new file.

**Comment by kfox1111** [comment]
Here's the isolated fix. Reverted back to the code that was reviewed by @faisal-memon and me.
https://github.com/spiffe/helm-charts/pull/239

We can then merge this on top to move the file around and get the better formatting.

**Comment by marcofranssen** [comment]
I don't get why you pr got reverted. I reviewed it as well. When I approved it merged and I figured all the patches where missing.

**Comment by kfox1111** [comment]
> I don't get why you pr got reverted. I reviewed it as well. When I approved it merged and I figured all the patches where missing.

It happened here at line 21

**Review by kfox1111 — APPROVED** [review]
Works locally. Looks ok.

**Comment by marcofranssen** [comment]
https://github.com/spiffe/helm-charts/pull/237/commits splitted the changes across 3 commits, still GitHub does not render the file as a rename due to the changes in the file.

To capture the specific changes in Git I splitted the commits, so `git blame`  can always show us why certain changes have been made.
